contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -210,35 +205,23 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     /**
      * @dev See {INativeTokenDestination-reportTotalBurnedTxFees}.
      */
-    function reportTotalBurnedTxFees(
-        TeleporterFeeInfo calldata feeInfo,
-        address[] calldata allowedRelayerAddresses
-    ) external {
-        uint256 adjustedFeeAmount;
-        if (feeInfo.amount > 0) {
-            adjustedFeeAmount = SafeERC20TransferFrom.safeTransferFrom(
-                IERC20(feeInfo.feeTokenAddress), feeInfo.amount
-            );
-        }
+    function reportBurnedTxFees(address[] calldata allowedRelayerAddresses) external {
+        uint256 burnedDifference = BURNED_TX_FEES_ADDRESS.balance - lastestBurnedFeesReported;
 
-        uint256 amount = BURNED_TX_FEES_ADDRESS.balance;
-        uint256 scaledAmount = _scaleTokens(amount, false);
+        uint256 scaledAmount = _scaleTokens(burnedDifference, false);
 
         bytes32 messageID = _sendTeleporterMessage(
             TeleporterMessageInput({
                 destinationBlockchainID: sourceBlockchainID,
                 destinationAddress: nativeTokenSourceAddress,
-                feeInfo: TeleporterFeeInfo({
-                    feeTokenAddress: feeInfo.feeTokenAddress,
-                    amount: adjustedFeeAmount
-                }),
-                requiredGasLimit: REPORT_BURNED_TOKENS_REQUIRED_GAS,
+                feeInfo: TeleporterFeeInfo({feeTokenAddress: address(0), amount: 0}),
+                requiredGasLimit: TRANSFER_NATIVE_TOKENS_REQUIRED_GAS,
                 allowedRelayerAddresses: allowedRelayerAddresses,
-                message: abi.encode(ITokenSource.SourceAction.Burn, abi.encode(scaledAmount))
+                message: abi.encode(GENERAL_BURN_ADDRESS, scaledAmount)
```

Comment:
don't we want this to go to the blackhole address so that the sources don't need specific handling of burn report transfers?

Reply:
`subnet-evm` actually disables transfers the the blackhole address. So while we could send ERC20s to that address, we couldn't send native tokens there.